glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -85,7 +85,10 @@ open class GleanInternalAPI internal constructor () {
         this.configuration = configuration
 
         this.gleanDataDir = File(applicationContext.applicationInfo.dataDir, GLEAN_DATA_DIR)
-        handle = LibGleanFFI.INSTANCE.glean_initialize(this.gleanDataDir.path, applicationContext.packageName, uploadEnabled.toByte())
+        handle = LibGleanFFI.INSTANCE.glean_initialize(
+            this.gleanDataDir.path,
+            applicationContext.packageName,
+            uploadEnabled.toByte())
```

Comment:
Maybe final `)` on the next line?  Maybe ktlint doesn't care, but I think that's more readable (and consistent with rustfmt FWIW).